Minor changes

cala opened the pull request:
Mana regeneration from spirit points
Unstealth when looting
Glancing blow fix

cala committed:
Modify the mana regeneration rate to be correct for 1.12 and below.

Mana from spirit should be per tick, each tick is every 2 seconds.

See:
http://www.wowwiki.com/Talk:Spirit#Spirit_versus_X_mana.2F5_seconds_item
s

cala committed:
Fix glancing blows formula for zero.

A difference in skill points should reward an extra 2% chance of
getting a glancing blow.

See: http://www.wowwiki.com/Glancing_blow#Glancing_Blow_chance

cala committed:
Remove Stealth when looting.

Stealth should be removed when looting:
http://www.wowwiki.com/Stealth

cala committed:
Change files permission.

Salja commented:
Jepp the mana fix is rigth i have sice 1 year https://github.com/Salja/MaNGOSZero/commit/b8da98d481623cfc01e23ba86de5d8564a1a4143

and the other all ok

Zakamurite commented on src/game/Player.cpp:
```diff
@@ -6937,6 +6937,9 @@ void Player::SendLoot(ObjectGuid guid, LootType loot_type)
 
     DEBUG_LOG("Player::SendLoot");
+	
+    RemoveSpellsCausingAura(SPELL_AURA_MOD_STEALTH);
```
Wrong way. Need use interrupt flag

cala replied:
Won't using interrupt flag also interrupt the player from looting corpse?
If not I will look how flag system work and try to do it.

cala replied:
Okay, I looked at the 25 available INTERRUPT_FLAG and none of them seem to be for loot. I understand what you suggest but everywhere else, stealth is removed this way: RemoveSpellsCausingAura(SPELL_AURA_MOD_STEALTH);
If you know which INTERRUPT_FLAG to use, I'll try it.

cala commented:
@Salja : my apologies, I didn't know who was the original author of this update I fetched from Picard64 repository.

Salja commented:
@cala it does not matter=) its fine wehn ppls post fixes for zero =) look in my repo i think you found some more good patches

cala commented:
@Salja ok, thank you. I'll look to your repo. =) I submitted those three ones because I was sure they were blizz-like.

cala committed:
Merge remote-tracking branch 'upstream/develop' into develop

KillaVulkan commented:
Yes, Salja repo is best!!!!!!
I used for my server :P

cala committed:
Improve interaction checks related to quests

Based on commit from mangos-one:
https://github.com/mangos-one/server/commit/12825ed1ba7bbb4a62d340d962f8
e8aadf08190a
'I see dead people, they are everywhere' - and now you can interact
with them for quests.
Please report any missing issues (could not test all cases)
Signed-off-by: Schmoozerd <schmoozerd@scriptdev2.com>

cala committed:
Fix Druid Enrage Armor. Thanks to Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/237b13a669f1094af91b4df08bdd6
d0813322d3c

cala committed:
Fix Shaman Spell Rockbiter Weapon. Patch by Zataron, Thanks Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/f6ef0b0b7d1cf3a28cbebdf3e15fa
d99bdc5a751

cala committed:
Fix Sheeped/Feared Targets Bugging. Thanks Zataron & Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/fb25ade7a13c04f7367e36c579ab2
beec02fb7ec

cala committed:
Lower Stealth Detection of Creatures. Thanks Zataron & Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/3e29bb5b9deb785a24bc511d052f8
b70fb0ceaa5

cala committed:
50% Chance for Rare Mobs to be Spawned on Server Start. Thanks Zataron & Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/5fdd2099e140bbfaf7f0da95608a6
4da8642e207

cala committed:
Fix Aggro issues of Npc Summons. Thanks Zataron & Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/08f43bef03ec261c26d7c5654f58a
975943f11fc

cala committed:
Fix Druid Attackpower in Cat and Bear Form. Patch by Zataron & Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/ae17557854aeeaeb34f4132376def
138ce3d788f

cala committed:
Fix Beastslayer Damage Bonus. Thanks Zataron & Salja

Based on commit from Salja/mangos-zero:
https://github.com/Salja/MaNGOSZero/commit/cdb4304d7ab28d076bf673992da42
e6c4a5c9c00

cala committed:
Breath/fatigue/fire damage timers tick each 2 sec.

Based on commit from mangos-one:
https://github.com/mangos-one/server/commit/6ab41a906e9f501b5399f15ca0b9
053788a36ccb
